core/src/commonTest/kotlin/performance/KordEventDropTest.kt

```diff
@@ -63,7 +68,9 @@ class KordEventDropTest {
     )
 
     @Test
-    fun `hammering the gateway does not drop core events`() = runBlocking {
+    @JsName("test1")
+    @IgnoreOnJs // currently this doesn't work on JS for some reason
+    fun `hammering the gateway does not drop core events`() = runTest {
```

Comment:
ok, it seems like the code is just too slow on js, is there a way to increase the timeout for kotlin js tests?

Reply:
yes you can, see https://github.com/kordlib/kord/pull/775/commits/7eb4a645503a49cdd2cdead337d02889147d2238